core/src/commonMain/kotlin/com/ustadmobile/core/view/PersonAccountEditView.kt

```diff
@@ -0,0 +1,33 @@
+package com.ustadmobile.core.view
+
+import com.ustadmobile.lib.db.entities.Person
+
+interface PersonAccountEditView: UstadEditView<Person> {
+
+    var firstPassword: String?
+
+    var secondPassword: String?
+
+    var secondPasswordFieldRequiredErrorVisible: Boolean
+
+    var firstPasswordFieldRequiredErrorVisible: Boolean
+
+    fun showPasswordDoNotMatchError()
+
+    var usernameRequiredErrorVisible: Boolean
+
+    var errorMessage:String?
+
+    var fistPasswordFieldHint: String?
```

Comment:
I think we should have three fixed password boxes : current password, new password, confirm new password. If the user is admin, we can hide the current password box.